frictionless/core.py

```diff
@@ -189,3 +192,15 @@ def resource_descriptor(dfile):
     return dict(**package_meta,
                 **{"resources": resources,
                    "created": datetime.now(timezone.utc).isoformat()})
+
+def build_datapackage_from_id(source: str) -> Dict:
+    """Build a datapackaage from a PUDL source."""
+    dp = {}
+    dp["profile"] = "data-package"
+    dp["homepage"] = "https://catalyst.coop/pudl/",
```

Comment:
Can we assume these will be the same for all datasets?

Reply:
The `profile` describes what kind of structures to expect within the data package, e.g. there are both tabular and genomic data package standards. But I think for our purposes it will only even be generic data packages.

As far as I know the PUDL homepage will apply to all of the data that we're archiving for the foreseeable future.